mora/service/orgunit.py

```diff
@@ -205,6 +205,16 @@ def get_one_orgunit(c, unitid, unit=None,
     elif details is UnitDetails.FULL:
         unittype = common.get_uuid(rels['enhedstype'][0], required=False)
 
+        if rels['overordnet'][0]['uuid'] is not None:
+            r['parent'] = get_one_orgunit(c, rels['overordnet'][0]['uuid'],
+                                          details=UnitDetails.FULL)
+
+            if r['parent'] is not None:
+                slash = '' if not r['parent']['path'] else '/'
+                r['path'] = r['parent']['path'] + slash + r['parent']['name']
```

Comment:
It just so happens that this statement has a quadratic execution time — but it [probably](https://stackoverflow.com/questions/4435169/how-do-i-append-one-string-to-another-in-python#4435194) won't matter…

That being said, this code region is a bit hard to read — I think it'd be cleaner if you used explicit `if`s for all three cases.